wunderio/next4drupal-project
# NEX-41 - Add authentication to the frontend

This PR adds:

* next auth setup, with default login form and configuration
* new user role to be assigned to users that are supposed to be able to log into the frontend
* edits to the consumer entity in drupal to allow people with this role to access drupal from the frontend
* a user menu block in the frontend, showing the name of the user and offering a link to log-out
* a user migration, adding 2 test users with the correct role
* new env vars for the login system <- Important for the silta setup
* a component to display parts of the page only to registered users



To test: 
1. run the usual big command to reset the environment
2. visit the frontend and log in using the test users: 
```
user: testuser1 pass: test
user: testuser2 pass: test
``` 
3. test that you can log out
4. test that the webform appears
5. test that you can submit the webform, and the result in drupal has the name of the logged-in user.
